modules/ROOT/pages/procedures.adoc

```diff
@@ -1125,6 +1125,44 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-ver
 | *Mode* 3+| DBMS
 |===
 
+[NOTE]
+====
+Users should prefer the usage of procedures.adoc#procedure_dbms_unquarantineDatabase[`dbms.unquarantineDatabase()`]
+over `dbms.quarantineDatabase()` because of the improvements and the new features (see the `operation` option ).
+The deprecated `dbms.quarantineDatabase()` procedure is available in Cypher 5 but not in Cypher 25.
+====
+
+[role=label--enterprise-edition label--admin-only]
+[[procedure_dbms_unquarantineDatabase]]
+=== dbms.unquarantineDatabase()
+
+
+.Details
+|===
+| *Syntax* 3+m| dbms.unquarantineDatabase(server, databaseName, operation) :: ()
+| *Description* 3+a| Remove the quarantine status from a database in a server performing a cleanup `operation`.
+The possible values for the cleanup operation are:
+
+* `keepStateKeepStore` -- do nothing; leave store and cluster state as they are.
+* `replaceStateKeepStore` -- join as a new member, clearing the current cluster state but keeping the store.
+* `replaceStateReplaceStore` -- join as a new member, clearing both the current cluster state and the store.
+.4+| *Input arguments* | *Name* | *Type* | *Description*
+| `server` | `STRING` | The name or the id of the server where there is the quarantined database.
+| `databaseName` | `STRING` | The name of the database whose quarantine status must be removed.
+| `operation` | `STRING` | Select a cleanup operation to perform while removing the quarantine.
+| *Mode* 3+| DBMS
+|===
+
+[NOTE]
+====
+If you choose to clear the current cluster state, the server will try to join as a new member,
+but this joining can succeed if and only if there is a majority of old members "letting" the
+new members in. Let's assume our cluster has a topology with three primaries. If there is only one server
+in `QUARANTINED` mode, then it is safe to choose `replaceStateKeepStore` or `replaceStateReplaceStore`.
+If there are two servers in `QUARANTINED` mode, then you should not use concurrently `replaceStateKeepStore` or
```

Comment:
```suggestion
If there are two servers in `QUARANTINED` mode, then you should not use concurrently `replaceStateKeepStore` or `replaceStateReplaceStore` for both servers because there would be no majority to let them in.
```